src/data/data_loader.py

```diff
@@ -40,10 +45,27 @@ def load(self):
                                train=True,
                                transform=self.transform(),
                                download=True)
-        return torch.utils.data.DataLoader(dataset=dataset,
+        if self.sampling_ratio == 1:
+            self.sampler = None
```

Comment:
Are we using this variable. Would make sense if we refactored to only have one return statement after the conditional statement. This could also improve readability

Reply:
Sure!